signup.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./signup.css" />
+    <title>BountyDo</title>
+</head>
+
+<body>
+    <header>
+        <div class="wrapper__logo">
+            <small class="bounty">BountyDo</small>
+        </div>
+
+        <div class="wrapper__buttons">
+            <a class="auth-buttons" href="signin.html">Sign In</a>
+        </div>
+
+    </header>
+
+    <div class="wrapper-signup">
+        <form action="signup.html">
+            <div class="main">
+                <h2 class="title">Sign Up</h2>
```

Comment:
I think we can create general file for same styles. For example `title` and `header` styles can be placed at it.